src/Native/Runtime/MathHelpers.cpp

```diff
@@ -82,7 +82,7 @@ EXTERN_C REDHAWK_API float REDHAWK_CALLCONV RhpFltRound(float value)
 
 #endif // !PROJECTN
 
-#ifdef _ARM_
+#if defined(_ARM_) || defined(_CROSS_TARGET_ARM_)
```

Comment:
I do not think we need a new `_CROSS_TARGET_ARM_` define for this. The existing `_ARM_`  should be used instead.

Reply:
You will get the x86 version of stackunwindind or exception handling in the runtime unless you define `_ARM_` properly.

Reply:
Since your steps include `Copy CoreRT framework and sdk binaries from target (armel) output folder`, why do you need this at all? You should be getting the proper armel lib.

Reply:
I am sorry, but I still do not understand.

Why do you need to build anything under `src/Native/Runtime/` at all during x86 crosstarget build? The x86 crosstarget build should be building the compiler only. All runtime bits should be comming from armel build.

Reply:
Ok, I get it. Perhaps this is because initially I tried to make the work completely with only x86. I.E. with the replacement only x86 target in rsp file from x86 to armel. But with some success there were some problems. And I return to combine x86+armel way. Apparently, therefore, there was such confusion. Ok, I'll see how to do it more properly.